dbt_subprojects/dex/models/_projects/bebop/arbitrum/bebop_jam_arbitrum_trades.sql

```diff
@@ -66,7 +66,7 @@ jam_unnested_array_taker AS (
         element_at(taker_tokens_raw, sequence_number) AS taker_token_address,
         element_at(taker_amounts_raw, sequence_number) AS taker_token_amounts,
         sequence_number - 1 AS taker_index
-    FROM bebop_jam_raw_data
+    FROM (select * FROM bebop_jam_raw_data WHERE taker_length > 0 AND maker_length > 0)
```

Comment:
Do you want to add the `WHERE` clause in the `bebop_jam_raw_data` CTE?

Reply:
@Hosuke done, thanks